config/set/laminas-servicemanager-40.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\LaminasServiceManager4\Rector\MethodCall\RemoveServiceLocatorRector;
+use Rector\Renaming\Rector\Name\RenameClassRector;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
```

Comment:
what about to add ChangePackageVersionComposerRector? O:-) see https://github.com/rectorphp/rector/blob/master/config/set/nette-31.php